src/components/Header.jsx

```diff
@@ -0,0 +1,11 @@
+import Button from "./Button";
+import Nav from "./Nav";
+
+export default function Header() {
```

Comment:
Any point in this component existing? Now it just displays the Nav and nothing else, seems like it's just an extra layer in the component tree for no reason.